src/daemon.cpp

```diff
@@ -29,6 +29,68 @@
 Q_LOGGING_CATEGORY(PACKAGEKITQT_DAEMON, "packagekitqt.daemon")
 Q_LOGGING_CATEGORY(PACKAGEKITQT_OFFLINE, "packagekitqt.offline")
 
+Q_DECLARE_METATYPE(PackageKit::PkPackage);
+Q_DECLARE_METATYPE(QList<PackageKit::PkPackage>);
+Q_DECLARE_METATYPE(PackageKit::PkDetail);
+Q_DECLARE_METATYPE(QList<PackageKit::PkDetail>);
+
+const QDBusArgument &operator<<(QDBusArgument &argument, const PackageKit::PkPackage &pkg){
```

Comment:
style: Opening bracket should be in its own line

Reply:
Addressed.

Reply:
Did you (force)push that change? Somehow I don't see it yet...

Reply:
It was, thanks! :-)